xonsh/dirstack.py

```diff
@@ -5,9 +5,12 @@
 from glob import iglob
 from argparse import ArgumentParser
 
+from xonsh.tools import get_sep
+
 DIRSTACK = []
 """A list containing the currently remembered directories."""
 
+
 def _get_cwd():
```

Comment:
It looks like it is still used a few other places via imports, so I don't think we should kill it.